supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -60,6 +170,30 @@ The AMD64 logo is trademarked; the term "AMD64" is not trademarked. For more inf
 For more information about Intel® trademarks, see the http://www.intel.com/content/www/us/en/legal/trademarks.html[Trademark Information] and http://www.intel.com/content/www/us/en/trademarks/trademarks.html[Usage Guidelines for Customers, Licensees, and Other Third Parties] pages.
 ====
 
+// AMQ: Kept as is, generic
+[discrete]
+[[jboss-amq]]
+==== image:images/yes.png[yes] AMQ (noun)
+*Description*: The short product name for Red Hat AMQ.
```

Comment:
The description refers to "Red Hat AMQ", but that term is also listed in the "Incorrect forms" section. Should one of these sections be updated?

Reply:
That is really odd, definitely seems wrong to me. I'm hesitant to make any changes outside of reorganizing the entries. 

I'll have to see what all comes up - maybe we can capture these types of things into an issue that people add to when they come across things, and we do a sweep after the reorg.

Reply:
@bredamc Based on the text of the Red Hat AMQ entry, I think it was an error to list "AMQ" in "Incorrect forms". The last time AMQ came up in Style Council, @bergerhoffer used someone named Fintan as a contact. Andrea, can Fintan assist again on this?